sensing/autoware_pointcloud_preprocessor/src/concatenate_data/concatenate_pointclouds.cpp

```diff
@@ -302,6 +314,51 @@ void PointCloudConcatenationComponent::publish()
   }
 }
 
+void PointCloudConcatenationComponent::publishSingleLidar()
+{
+  stop_watch_ptr_->toc("processing_time", true);
+  sensor_msgs::msg::PointCloud2::SharedPtr transformed_cloud_ptr(
+    new sensor_msgs::msg::PointCloud2());
+  if (cloud_stdmap_.cbegin()->second) {
+    static_tf_buffer_->transformPointcloud(
+      this, output_frame_, *cloud_stdmap_.cbegin()->second, *transformed_cloud_ptr);
+  }
+
+  // publish concatenated pointcloud
+  if (transformed_cloud_ptr) {
+    auto output = std::make_unique<sensor_msgs::msg::PointCloud2>(*transformed_cloud_ptr);
+    pub_output_->publish(std::move(output));
+  } else {
+    RCLCPP_WARN(
+      this->get_logger(), "transformed_cloud_ptr is nullptr, skipping pointcloud publish.");
+  }
+
+  updater_.force_update();
+
+  // add processing time for debug
+  if (debug_publisher_) {
+    const double cyclic_time_ms = stop_watch_ptr_->toc("cyclic_time", true);
+    const double processing_time_ms = stop_watch_ptr_->toc("processing_time", true);
+    debug_publisher_->publish<tier4_debug_msgs::msg::Float64Stamped>(
+      "debug/cyclic_time_ms", cyclic_time_ms);
+    debug_publisher_->publish<tier4_debug_msgs::msg::Float64Stamped>(
+      "debug/processing_time_ms", processing_time_ms);
+  }
+  for (const auto & e : cloud_stdmap_) {
+    if (e.second != nullptr) {
+      if (debug_publisher_) {
+        const auto pipeline_latency_ms =
+          std::chrono::duration<double, std::milli>(
+            std::chrono::nanoseconds(
+              (this->get_clock()->now() - e.second->header.stamp).nanoseconds()))
+            .count();
+        debug_publisher_->publish<tier4_debug_msgs::msg::Float64Stamped>(
+          "debug" + e.first + "/pipeline_latency_ms", pipeline_latency_ms);
+      }
+    }
+  }
```

Comment:
Same proposal as [this](https://github.com/autowarefoundation/autoware.universe/pull/8535/files#r1746771145).